### ISSUE TITLE
How to color the detected planes of an object with different color with open 3d library

### ISSUE THREAD

**Issue by floridak** [isl-org/Open3D#4654]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### My Question

if i had those lines : plane = np.concatenate([point, [cluster_size]])
planes_out.append(plane)
planes_out = np.array(planes_out)
so basically a lot of planes, how would I color them with different color? can you plzz have a look? thanks in advance

**Comment by theNded** [comment]
You can generate a mesh per plane (standard 4 vertices, 2 triangles) and use `paint_uniform_color` to assign colors.

**Comment by floridak** [comment]
But the planes are already there in np arrays. Why do we need to create the mesh per each plane?
